src/components/table.js

```diff
@@ -746,16 +741,30 @@ export const ColumnSettings = ({ columnSettings, onChange }) => {
               h(SortableHandleDiv, { style: styles.columnHandle }, [
                 icon('columnGrabber', { style: { transform: 'rotate(90deg)' } })
               ]),
-              div({ style: { display: 'flex', alignItems: 'center' } }, [
-                h(LabeledCheckbox, {
-                  checked: visible,
-                  onChange: () => toggleVisibility(index)
+              h(IdContainer, [id => h(Fragment, [
+                h(TooltipTrigger, {
+                  // Since entity names don't contain spaces, word-break: break-all is necessary to
```

Comment:
nit: should this comment read "Since attribute names..." and "...wrap the attribute name..." instead?

Reply:
Yes. Fixed in https://github.com/DataBiosphere/terra-ui/pull/2941/commits/a6512a11293a66b9fe936fb9ce055888291ccc61.